src/sass/index.css

```diff
@@ -0,0 +1,229 @@
+.dropdown-kebab-pf.btn-group {
```

Comment:
Please remove the whole `/src/sass` directory. In master, all sass files are in the `sass` directory (sibling of src)